src/containers/DownloadCsv/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import fetchCsv from '../../data/actions/courseEnrollments';
+import DownloadCsv from '../../components/DownloadCsv';
+
+const mapStateToProps = state => ({
```

Comment:
The `mapStateToProps` for this container should have a similar implementation to [how it's done for the table](https://github.com/edx/edx-portal/blob/8b672d5975aeeb94237cf1ea7f46f6bba4fef4c9/src/containers/TableContainer/index.jsx#L6), where all download buttons' loading/error states will be contained in a single place in the Redux store. 

The resulting store should look something like:
```javascript
{
  ...
  table: {
    enrollments: { ... },
   ...
  },
  csv: {
    enrollments: {
      error: null,
      loading: false,
    }
    ...
  }
}
```